tps22994/tps22994.h

```diff
@@ -5,10 +5,11 @@
 #include <nrf_drv_twi.h>
 
 // Address is actually 0xE0 but only the 7 LSB counts. (bangs head into desk)
+//E0 E2 E4 E6 E8 EA EC
```

Comment:
Isn't it obvious ? It's a list of the potential TPS I2C addresses :) No need to document this, since it will be hard wired, so I removed the comment.